freud/density.pyx

```diff
@@ -732,3 +733,126 @@ cdef class RDF(_SpatialHistogram1D):
             return freud.plot._ax_to_bytes(self.plot())
         except (AttributeError, ImportError):
             return None
+
+
+cdef class ContinuousCoordination(_PairCompute):
+    r"""Computes the continuous local coordination number.
+
+    The :class:`ContinuousCoordination` impliments extensions of the Voronoi
+    discrete coordination number to the real numbers. These are motivated by the
+    dissertation <cite>. The formulas for the various implementations are:
+
+    Power
+
+    .. eq::
+
+        CN_p = N^{2.0 - m} \sum_{i=1}^{k}
+        {\left[\left(\frac{V_i}{V}\right)^{m}\right]}^{-1}
+
+    Log
+
+    .. eq::
+
+        CN_{log} = \frac{-1}{\log{N}} \sum_{i=1}^{k}\log{\left(\frac{V_i}{V}\right)}
+
+    Exp
+
+    .. eq::
+
+        CN_{exp} = \sum_{i=1}^{k}\exp{\left(\frac{V_i}{V} - \frac{1}{N} \right)}
+
+    where :math:`k` is the number of neighbors a particle has, :math:`V_i` is
+    the volume of the pyramid (or area of the triangle) whose base is the
+    Voronoi polytope facet between the central particle and neighobor :math:`i`
+    and whose height is half the distance vector, and :math:`V` is the
+    volume/area of the Voronoi polytope.
+
+    Args:
+        powers (list[float]): The powers to compute the continuous coordination
+            number for.
+        compute_log (`bool`, optional): Whether to compute the log continuous
+            coordination number.
+        compute_exp (`bool`, optional): Whether to compute the exp continuous
+            coordination number.
```

Comment:
I am not sure what the _correct_ default is. A user can easily do what you are saying by `ContinuousCoordination([], True)` for instance. I can set the default value to `()` which would be as unopinionated as possible.

Reply:
That works for me. I am asking you for the most reasonable set of defaults because you have the most experience with this compute and probably know which coordinations are more/less useful.